environments/common/security-group/security-group.tf

```diff
@@ -31,16 +32,22 @@ resource "aws_security_group" "alb_security_group" {
 /* Security group for the ecs application */
 resource "aws_security_group" "ecs_security_group" {
   name        = "trade-tariff-ecs-security-group-${var.environment}"
-  description = "Allow TLS ingress, all egress"
+  description = "Allow HTTP/S ingress, all egress"
   vpc_id      = data.terraform_remote_state.base.outputs.vpc_id
 
   # ingress traffic from alb
   ingress {
-    description     = "HTTPS Access From ALB"
-    from_port       = 443
-    to_port         = 8080
-    protocol        = "tcp"
-    security_groups = [aws_security_group.alb_security_group.id]
+    description = "HTTP Access from ALB"
+    from_port   = 80
+    to_port     = 8080
+    protocol    = "tcp"
+  }
+
+  ingress {
```

Comment:
Do you still need 443 from the ALB?

Reply:
In theory, no, but I'm going to leave it open until it's absolutely confirmed we're not doing any traffic on 443